src/com/intellij/idea/plugin/hybris/tools/remote/console/impl/HybrisGroovyConsole.kt

```diff
@@ -83,6 +83,10 @@ class HybrisGroovyConsole(project: Project) : HybrisConsole(project, HybrisConst
 
     override fun icon(): Icon = JetgroovyIcons.Groovy.Groovy_16x16
 
+    fun updateCommitMode(commitMode: Boolean) {
```

Comment:
modification of the state can be replaced with just two action buttons for Commit ON and Commit OFF.

it can be done a separate PR

Reply:
We need to discuss it.